streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorRecordContext.java

```diff
@@ -90,7 +90,10 @@ public long sizeBytes() {
         if (headers != null) {
             for (final Header header : headers) {
                 size += header.key().toCharArray().length;
```

Comment:
Add a check here that `header.key()` doesn't return `null`?

Reply:
The key cannot be `null` -- compare https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/header/internals/RecordHeader.java

Reply:
Thanks for the clarification.